src/package5_1/package2/Class3.java

```diff
@@ -0,0 +1,4 @@
+package package5_1.package2;
+
+public class Class3 {
+}
```

Comment:
Why is this method empty?

Reply:
because we needed to create similar classes in different packages, nothing was said about methods